### DISCUSSION TITLE
Welcome to Robinhood API Documentation Discussions!

### DISCUSSION THREAD

**Discussion by sanko** [sanko/Robinhood#92] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by askarovoren** [comment]
Is there a way to create an SSO to Robinhood platform using a 3rd party web application using the API?

**Reply by sanko** [reply]
No.

**Comment by wlinInspire** [comment]


**Reply by sanko** [reply]
Yes.

**Reply by sanko** [reply]
No.

**Comment by swimclan** [comment]
I assume the reason why Cloudfront is blocking my calls to the auth endpoint (404) is because I am making the call from the US?

**Comment by Spencerg00dwin** [comment]
Does the auth on this repo still work?
